CHANGELOG.md

```diff
@@ -97,6 +97,7 @@
 
 ### Mimirtool
 
+* [ENHANCEMENT] Added `mimirtool alertmanager verify` command to validate configuration without uploading. #3247
```

Comment:
[nit] I think you can proudly call it a new feature ;) We're also used to reference the PR number, so #3440 in this case.

```suggestion
* [FEATURE] Added `mimirtool alertmanager verify` command to validate configuration without uploading. #3440
```

Reply:
Done. :)